lib/dest/writeContents/index.js

```diff
@@ -60,17 +60,16 @@ function writeContents(writePath, file, cb) {
   }
 
   function isErrorFatal(err) {
+    var isFatal = true;
```

Comment:
This was some weird rule about an `else` after a return statement. I don't remember seeing this warning before though in the other repo's. Seems like a funny rule since your code was legit, but I can see how it could get confusing in other complex cases.